Project/js/menu.js

```diff
@@ -0,0 +1,5 @@
+function menuOpen(){
```

Comment:
maybe you could cache those elements (assign them to variables)? it's more efficient when user clicks many times on that button